lisa/sut_orchestrator/libvirt/ch_platform.py

```diff
@@ -113,9 +114,21 @@ def _create_node_domain_xml(
 
         os_type = ET.SubElement(os, "type")
         os_type.text = "hvm"
-
         os_kernel = ET.SubElement(os, "kernel")
-        os_kernel.text = node_context.firmware_path
+        if node_context.guest_vm_type == "ConfidentialVM":
+            os_kernel.text = "/usr/share/cloud-hypervisor/cvm/linux.bin"
```

Comment:
Can we make it runbook parameter. If nothing set from runbook, we can use this as default path.

Reply:
Sure, we can do that.